packages/sitecore-jss/src/personalize/utils.test.ts

```diff
@@ -12,26 +12,35 @@ import {
 
 describe('utils', () => {
   describe('getPersonalizedRewrite', () => {
-    const data = {
-      variantId: 'variant-id',
-    };
+    const variantIds = ['123', '456'];
```

Comment:
I feel like it's best to have variant with underscore here, just in case, to catch any weirdness with component variants if it happens

Reply:
Good idea. Added.